### PR TITLE
Fixed formatText method ambiguity, added custom attribute to show text pre-formatting.

### PR DESCRIPTION
**Pull request by Josvanegmond**

### CODE REVIEW

File: src/com/egeniq/widget/Button.java [hunk 1]
```diff
@@ -50,11 +52,20 @@ public Button(Context context, AttributeSet attrs, int defStyle) {
...
         }
 
-        _init();
+        _init(context, attrs);
     }
 
-    private void _init() {
+    private void _init(Context context, AttributeSet attrs) {
         _originalText = getText();
+        Log.d("button", _originalText.toString());
```
Comment (insomnaicJustin) [inline src/com/egeniq/widget/Button.java]: this should be only in debug, right?

Comment (Josvanegmond) [inline src/com/egeniq/widget/Button.java]: correct, removing

**Comment by Josvanegmond** [comment]
Update

**Comment by insomnaicJustin** [comment]
reviewed, don't have rights to merge

**Pull request merged**